### PR TITLE
Minor tweaks

### PR DESCRIPTION
**Pull request by Richienb**

### CODE REVIEW

**Comment by sindresorhus** [comment]
If you wanna do some other improvements, you could also turn the regex groups into named groups. And `x.match(/\S+/g)` should use `.exec` instead.

File: index.js [hunk 1]
```diff
@@ -26,52 +26,42 @@ const win32 = async () => {
...
-		// TODO: Remove the `.reduce`.
-		// eslint-disable-next-line unicorn/no-reduce
-		.reduce((result, x) => {
-			if (isProtocol(x)) {
-				result.push(x.match(/\S+/g) || []);
-			}
-
-			return result;
-		}, []);
+	return list.split('\n').filter(item => isProtocol(item)).map(item => /\S+/g.exec(item) || []);
```
Comment (sindresorhus) [inline index.js:58]:
```suggestion
	return list
		.split('\n')
		.filter(item => isProtocol(item))
		.map(item => /\S+/g.exec(item) || []);
```

for readability?

**Comment by sindresorhus** [comment]
Tests are failing

File: index.js [hunk 2]
```diff
@@ -26,52 +26,45 @@ const win32 = async () => {
...
+		return Number.parseInt(groups.pid, 10);
 	}
-
-	return Number.parseInt(match[1], 10);
 };
 
 const getPort = (port, list) => {
 	const regex = new RegExp(`[.:]${port}$`);
-	const foundPort = list.find(x => regex.test(x[columns[0]]));
+	const foundPort = list.find(value => regex.test(value[addressColumn]));
```
Comment (sindresorhus) [inline index.js:46]: Maybe we could use a more specific name than `value`.

File: index.js [hunk 3]
```diff
@@ -26,52 +26,45 @@ const win32 = async () => {
...
-			if (isProtocol(x)) {
-				result.push(x.match(/\S+/g) || []);
-			}
-
-			return result;
-		}, []);
+	return list
+		.split('\n')
+		.filter(item => isProtocol(item))
+		.map(item => item.match(/\S+/g) || []);
```
Comment (Richienb) [inline index.js:61]: In this scenario, using `.match` is the most optimal way to solve this problem. Using `.exec` would open a whole new can of worms that we don't need to deal with.

**Comment by sindresorhus** [comment]
Thanks. Nice work as always :)

**Pull request merged**